### PR TITLE
Updated read query for updated database.

### PR DESCRIPTION
**Pull request by codyhho**

### CODE REVIEW

**Comment by davisethan** [comment]
I only see your .gitignore changed

**Comment by codyhho** [comment]
Hey Ethan there should be a second page that you can check out with the query that shows this:

`SELECT cart_items.cart_item_id AS item_id, items.item_name AS name, GROUP_CONCAT(DISTINCT cart_item_tags.cart_item_tag_name SEPARATOR ', ') AS tags, cart_items.quantity from cart_items
JOIN items on cart_items.cart_item_id = items.item_id
JOIN cart_item_tags on items.item_id = cart_item_tags.cart_item_id
WHERE cart_items.user_id = ?
GROUP BY items.item_id, cart_items.quantity;

File: server/queries.js [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
-FROM users_items
-JOIN items ON users_items.item_id = items.item_id
-JOIN items_tags ON items.item_id = items_tags.item_id
-JOIN tags ON items_tags.tag_id = tags.tag_id
-WHERE items_tags.is_cart_tag = 1 AND users_items.user_id = ?
-GROUP BY items.item_id, users_items.quantity`;
+
+exports.cartRead = `SELECT cart_items.cart_item_id AS id, items.item_name AS name, GROUP_CONCAT(DISTINCT cart_item_tags.cart_item_tag_name SEPARATOR ', ') AS tags, cart_items.quantity from cart_items
+JOIN items on cart_items.cart_item_id = items.item_id
+JOIN cart_item_tags on items.item_id = cart_item_tags.cart_item_id
```
Comment (davisethan) [inline server/queries.js:8]: Should be `JOIN cart_item_tags ON cart_items.cart_item_id = cart_item_tags.cart_item_id` ?

Comment (codyhho) [reply server/queries.js:8]: Thanks ethan! I did not catch that and thought it was working, revising now.

File: server/queries.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 exports.cartRead = `SELECT cart_items.cart_item_id AS id, items.item_name AS name, GROUP_CONCAT(DISTINCT cart_item_tags.cart_item_tag_name SEPARATOR ', ') AS tags, cart_items.quantity from cart_items
 JOIN items on cart_items.cart_item_id = items.item_id
```
Comment (davisethan) [inline server/queries.js:7]: Should be `JOIN items ON cart_items.item_id = items.item_id`

**Review by davisethan — APPROVED** [review]

**Pull request merged**